source/app/requesthandlers/eventbus_request_handler.py

```diff
@@ -31,9 +29,11 @@ def __init__(self, event, context):
 
         # Setup logging
         self._is_trace_enabled = os.getenv("TRACE", False)
-        logging_stream_name = "-".join([LOG_STREAM_PREFIX])
+        logging_stream_name = "-".join(["eventbus_request_handler"])
         dt = datetime.utcnow()
-        logstream = LOG_STREAM.format(logging_stream_name, dt.year, dt.month, dt.day)
+        logstream = "{}-{:0>4d}{:0>2d}{:0>2d}".format(
+            logging_stream_name, dt.year, dt.month, dt.day
+        )
```

Comment:
maybe I'm too function-happy, but I typically would lean toward including something like this as a separately named function entirely. This works fine as is, but I would be curious as to the team's opinion on code style best practices for stuff like this